app/src/main/java/app/revanced/tiktok/settingsmenu/preference/categories/FeedFilterPreferenceCategory.java

```diff
@@ -0,0 +1,54 @@
+package app.revanced.tiktok.settingsmenu.preference.categories;
+
+import android.content.Context;
+import android.preference.PreferenceScreen;
+import app.revanced.tiktok.settings.SettingsEnum;
+import app.revanced.tiktok.settingsmenu.SettingsStatus;
+import app.revanced.tiktok.settingsmenu.preference.RangeValuePreference;
+import app.revanced.tiktok.settingsmenu.preference.TogglePreference;
+
+@SuppressWarnings("deprecation")
+public class FeedFilterPreferenceCategory extends ConditionalPreferenceCategory {
+    public FeedFilterPreferenceCategory(Context context, PreferenceScreen screen) {
+        super(context, screen);
+
+
+        setTitle("Feed filter");
+
+        addPreference(new TogglePreference(
+                context,
+                "Remove feed ads", "Remove ads from feed.",
+                SettingsEnum.REMOVE_ADS
+        ));
+        addPreference(new TogglePreference(
+                context,
+                "Hide livestreams", "Hide livestreams from feed.",
+                SettingsEnum.HIDE_LIVE
+        ));
+        addPreference(new TogglePreference(
+                context,
+                "Hide story", "Hide story from feed.",
+                SettingsEnum.HIDE_STORY
+        ));
+        addPreference(new TogglePreference(
+                context,
+                "Hide image video", "Hide image video from feed.",
+                SettingsEnum.HIDE_IMAGE
+        ));
+        addPreference(new RangeValuePreference(
+                context,
+                "Hide view count", "If app show error, please change value or refresh few times.",
+                SettingsEnum.HIDE_PLAY_COUNT
+        ));
+        addPreference(new RangeValuePreference(
+                context,
+                "Hide like count", "If app show error, please change value or refresh few times.",
```

Comment:
What does this description mean? Please explain it if possible

Reply:
Sometime, when I set the min like count to high like 1M or higher tiktok app will show error because the current fetched feed list don't have any video which have like count higher than 1M. If that happen, I have to pull down, press the refresh button , restart app or lower the value. It will work fine when tiktok app have fetched enough video that at least one of them meet the filter user set.

Reply:
Can you fix this case's preference titles and descriptions to match your description?

Reply:
I think about "Set view number range" and "Set like number range" for the titles and keep the descriptions. Is it suitable?

Reply:
How about "Min likes" and "Min views" accompanied by a description?

Reply:
![image](https://github.com/ReVanced/revanced-integrations/assets/43563783/c12c6b51-80be-4090-b66e-026fd36c1690)
That settings let us change the min and max like the image. So I think "Min likes" is not really suitable.

Reply:
Min/Max likes and Min/max views?